src/content/ja/updates/2017/11/overscroll-behavior.md

```diff
@@ -0,0 +1,249 @@
+
+<figure class="clearfix centered">
+  <div class="attempt-left">
+    <video src="/web/updates/images/2017/11/overscroll-behavior/chatbox-double-refresh.mp4" autoplay muted loop height="225"></video>
+    <figcaption>前</figcaption>
+  </div>
+  <div class="attempt-right">
+    <video src="/web/updates/images/2017/11/overscroll-behavior/chatbox-double-refresh-fix.mp4" autoplay muted loop height="225"></video>
+    <figcaption>後</figcaption>
+  </div>
+</figure>
+
+[完全なコードの](https://github.com/ebidel/demos/blob/master/chatbox.html)スニペットは次のとおり[です](https://github.com/ebidel/demos/blob/master/chatbox.html) 。
+
+```html
+<style>
+  body.refreshing #inbox {
+    filter: blur(1px);
+    touch-action: none; /* prevent scrolling */
+  }
+  body.refreshing .refresher {
+    transform: translate3d(0,150%,0) scale(1);
+    z-index: 1;
+  }
+  .refresher {
+    --refresh-width: 55px;
+    pointer-events: none;
+    width: var(--refresh-width);
+    height: var(--refresh-width);
+    border-radius: 50%; 
+    position: absolute;
+    transition: all 300ms cubic-bezier(0,0,0.2,1);
+    will-change: transform, opacity;
+    ...
+  }
+</style>
+
+<div class="refresher">
+  <div class="loading-bar"></div>
+  <div class="loading-bar"></div>
+  <div class="loading-bar"></div>
+  <div class="loading-bar"></div>
+</div>
+
+<section id="inbox"><!-- msgs --></section>
+
+<script>
+  let _startY;
+  const inbox = document.querySelector('#inbox');
+
+  inbox.addEventListener('touchstart', e => {
+    _startY = e.touches[0].pageY;
+  }, {passive: true});
+
+  inbox.addEventListener('touchmove', e => {
+    const y = e.touches[0].pageY;
+    // Activate custom pull-to-refresh effects when at the top of the container
+    // and user is scrolling up.
+    if (document.scrollingElement.scrollTop === 0 && y > _startY &&
+        !document.body.classList.contains('refreshing')) {
+      // refresh inbox.
+    }
+  }, {passive: true});
+</script>
+```
+
+## オーバースクロールグローとラバーバンディング効果の無効化{: #disableglow }
+
+スクロール境界にヒットしたときにバウンス効果を無効にするには、 `overscroll-behavior-y: none`使用します。
+
+```css
+body {
+  /* Disables pull-to-refresh and overscroll glow effect.
+     Still keeps swipe navigations. */
+  overscroll-behavior-y: none;
+}
+```
+
+<figure class="clearfix centered">
+  <div class="attempt-left">
+    <video src="/web/updates/images/2017/11/overscroll-behavior/drawer-glow.mp4" autoplay muted loop height="300" class="border"></video>
+    <figcaption><b>変更前</b> ：スクロール境界を押すとグローが表示されます。</figcaption>
+  </div>
+  <div class="attempt-right">
+    <video src="/web/updates/images/2017/11/overscroll-behavior/drawer-noglow.mp4" autoplay muted loop height="300" class="border"></video>
+    <figcaption><b>後</b> ：グローが無効になりました。</figcaption>
+  </div>
+</figure>
+
+注：これにより、左/右のスワイプナビゲーションが保持されます。ナビゲーションを防ぐには、 `overscroll-behavior-x: none`使用できます。ただし、これは[まだ](https://crbug.com/762023) Chrome [に実装さ](https://crbug.com/762023)れています。
```

Comment:
https://github.com/gitlocalize/feedback/issues/67

Reply:
@agektmr We'll implement this, but it would be better if you could recreate this issue in the following repository:

https://github.com/alconost/gitlocalize-google

we use to communicate about Google Web Fundamentals.